pages/glossary/visual-reference-words.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Visual Reference Words
+key: visual-reference-words
+unambiguous: true
+objective: false
+input_aspects:
+  - Language
+---
+
+Any word in a text node that is included in the translated version of the following lists, where the language of the translation is the programmatically determinable language of the text node. Some words can be translated in multiple ways or have plural forms. In such cases each translation or form must be included. Some words can be spelled in multiple ways (For example: sometimes the word is capitalized and sometimes it isn't). In such cases each spelling must be included.
```

Comment:
Should there be test cases using different languages?

Reply:
Very good question…
I don't know. I am not sure this really add that much (the same way that there is not one example for each word, or that rules "foo is descriptive" usually only have English examples)… OTOH, it's easy to add…

Maybe a good example would be using some faux amis where the word would be a visual reference word in one language but not in the other and then the result of the rule would depend on the language of the page… But that is quite harder to find (no example from the top of my head) (one example would be "square" which is used in French with the meaning of "small garden or place" while the 4-side figure is "carré", but that is close to the similar meaning in English, although English seems to reserve it for larger space (town square, Traflagar square, …)). 
Otherwise, I'm not sure that "cliquer sur le bouton bleu" brings much more than "click the blue button".

(also realising that the lang attribute is missing on all examples 😖 )

@ShadowBB @carlosapaduarte @daniel-montalvo Do you have an opinion on that?

Reply:
I've tried an example where language matters. Let me know what you think about it…